src/styles/_variables.scss

```diff
@@ -66,6 +66,8 @@ $card-shadow: $shadow;
 
 // navbar
 $navbar-item-hover-background-color: #e9ecef;
+$navbar-height: 5.25rem; // 84px
```

Comment:
This variable is defined at Bluma.io, if you overwrite this variable, it will be applied to global `navbar-height`, at this time, the self-defined height does not take into effect. 

So you might need to change the name of `$navbar-height`